### ISSUE TITLE
브랜치 정리

### ISSUE THREAD

**Issue by hoony-o-1** [codeforseoul/congress-report#3]
@shindoh @appetizermonster 브랜치를 정리하면 좋을 것 같은데 일단 제가 선호하는 스타일은 master를 1.0 버전을 기준으로 두고 개발을 develop 같은 이름으로 진행하는 걸 좋아해요. 그리고 기능에 따라 브랜치를 나누는 것도 필요에 따라? 현재는 attend_api 브랜치로 작업이 되고 있는데 현재 이 레포지토리에서는 어떤 작업을 진행하고 계시나요? 아마 데이터를 크롤해오는 부분. 그리고 메일을 만들기 위해 데이터를 조합하는 부분일텐데.....

아 생각해보니 데이터를 크롤해오는 부분이 동헌님, 희진님이 하시고 메일을 쏘기위해 데이터를 사용하는 파트는 메일러를 작업하는 저랑 주희님 파트일까요? 그렇담 방금 커밋한 mailer.py는 지워야겠는데 댓글 부탁드려요! :+1: 

결국 쓰다보니 제가 잘못 커밋한 기분....

**Comment by shindoh** [comment]
넵. 제 담당은 말씀하신것이 맞습니다. 
 attend_api에서는 데이터를 크롤하고 정재하여 결과 데이터를 리턴해주는 소스를 작성하고 있습니다. 회의 참석 정보를 크롤해서 가져오는 것은 몇주전에 되었지만 일이 갑자기 바빠지는 바람에 정재하는 코드는 작성을 아직 못하였습니다. 저희가 구상한 서비스가 실시간으로 제공되어야 하는 정보가 아니기에 디비를 이용하기보다는 pandas를 이용해서 하려고 하고 있습니다. 일때문에 계속 일정이 딜레이 되는것은 죄송하지만 제 담당부분은 제가 마무리 지었으면 합니다. 결과 데이터를 받는 부분은 임의로 데이터를 모델링하시고 작업하시면 제가 그 모델에 맞추어 api를 작성하겠습니다. 
감사합니다 :)

**Comment by hoony-o-1** [comment]
네! 너무 죄송해하실 것은 없어요~! 시간 되실 때 한줄씩이라도 커밋해주시면 감사합니다 :) 지금 각 파트 담당이
- @dongdong 님, @yejiyun 님이 웹페이지
- @shindoh @appetizermonster 님이 데이터 크롤 및 정제
- @JH27 님께서 정제된 데이터로 메일 템플릿 작성

이렇게 작업이 분배되어 있어요. 참고해주세요! :+1: 

그리고 언제가 될지는 모르지만 영택님께서 코드포서울 데이터포탈을 올릴 예정이세요. 그렇게 되면 정제된 데이터를 메일 쏠 때만 이용하는 것이 아니라 쏠 때마다 차곡차곡 쌓아두는 것도 의미가 있을 것 같아요.
